pulsar-broker/src/main/java/org/apache/pulsar/EmbeddedPulsar.java

```diff
@@ -0,0 +1,139 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar;
+
+import org.apache.pulsar.broker.ServiceConfiguration;
+import org.apache.pulsar.broker.ServiceConfigurationUtils;
+
+import static org.apache.commons.lang3.StringUtils.isBlank;
+
+public class EmbeddedPulsar extends StandalonePulsarBase {
+
+    private EmbeddedPulsar(){}
+
+    public static EmbeddedPulsarBuilder builder(){
+        return EmbeddedPulsarBuilder.instance();
+    }
+
+    public static void main(String[] args) throws Exception {
```

Comment:
not sure we need a main function here. I think the purpose of this class is to have a `Builder` to build a standalone cluster, so application can use it for testing. How application start the standalone cluster is up to application.

Reply:
they can do that this is code example.